services/page-state.js

```diff
@@ -139,16 +139,16 @@ function getPageState() {
 
 /**
  * update publish button when scheduled
- * @param {boolean} isScheduled
+ * @param {string} state
+ * @param {boolean} enabled
  */
-function toggleScheduled(isScheduled) {
-  var el = document.querySelector('.kiln-toolbar-inner .publish'),
-    scheduledClass = 'scheduled';
+function toggleButton(state, enabled) {
+  var el = dom.find('.kiln-toolbar-inner .publish');
 
-  if (isScheduled) {
-    el.classList.add(scheduledClass);
+  if (enabled) {
+    el.classList.add(state);
```

Comment:
you could replace these lines with `el.classList.togger(state, enabled)` https://developer.mozilla.org/en-US/docs/Web/API/Element/classList